scripts/utils.py

```diff
@@ -384,6 +435,107 @@ async def call(contract_name, function_name, *inputs, address=None):
     return await contract.functions[function_name].call(*inputs)
 
 
+def get_contract(contract_path: str) -> Web3Contract:
+    path = os.path.splitext(contract_path)[0]
+    folder, contract_name = os.path.split(path)
+    compilation_output = json.load(
+        open(
+            Path(folder) / "build" / f"{contract_name}.sol" / f"{contract_name}.json",
+            "r",
+        )
+    )
+    return Web3().eth.contract(
+        abi=compilation_output["abi"], bytecode=compilation_output["bytecode"]["object"]
+    )
+
+
+def get_contract_deployment_bytecode(contract: Web3Contract):
+    return contract.bytecode
+
+
+def get_contract_method_calldata(
+    contract: Web3Contract, method_name: str, *args, **kwargs
+):
+    return contract.get_function_by_name(method_name)(
+        *args, **kwargs
+    )._encode_transaction_data()
```

Comment:
ideally we'd want to have something like in the test, there is one utils like `get_solidity_contract()` and we can use its output as in the tests